app/src/userpages/components/PurchasesPage/index.jsx

```diff
@@ -59,6 +63,21 @@ const PurchasesPage = () => {
             })
     }, [dispatch, filter])
 
+    useEffect(() => {
+        dispatch(getAllCommunityStats())
+    }, [dispatch])
+
+    const members = useMemo(() => (
+        (communityStats || {}).reduce((result, { id, memberCount }) => {
+            if (!memberCount) { return result }
+
+            return {
+                ...result,
+                [id.toLowerCase()]: memberCount.total,
+            }
+        }, {})
+    ), [communityStats])
```

Comment:
Seeing this piece of code twice in this PR. Can we have a helper function or a hook for it?

Reply:
It was used actually three times, added a hook 🎉